api/models/article.js

```diff
@@ -1,53 +1,101 @@
+import { ObjectId } from "mongodb";
 import mongoose from "mongoose";
-import ArticleStatus from "./helper.js";
+import ArticleStatus from "./enums/article_status";
+import WritingStatus from "./enums/writing_status";
+import InternalCommentStatus from "./enums/internal_comment_status";
+import DesignStatus from "./enums/design_status";
+import PhotographyStatus from "./enums/photography_status";
+
 const Schema = mongoose.Schema;
-const model = mongoose.model;
 
-/**
- * Approved Article Schema
- */
+//article schema
 const ArticleSchema = new Schema(
   {
-    title: String,
-    author: String,
-    authorUsername: String,
-    year: Number,
-    major: String,
-    categories: [{ type: String }],
-    date: Date,
-    coverImage: String,
-    images: [{ type: String }],
-    body: [{ type: String }],
-    pullquotes: [{ type: String }],
-    sources: [{ type: String }],
-    comments: [
+    title: { type: String, required: true },
+    authors: [{ type: ObjectId, required: true }],
+    categories: { type: [String], required: true },
+    sources: { type: [String], required: true },
+    ArticleContent: [
       {
-        author: String,
-        date: Date,
-        body: String,
+        contentType: { type: String, enum: ["PullQuote", "body", "image"], required: true },
```

Comment:
Article Content should be a mixed type of 3 schemas, PullQuote, BodyParagraph, or Image, not a String that is of an enum value.

Note that this is NOT a linked object, but rather a SubDocument.